sympy/integrals/prde.py

```diff
@@ -1038,6 +1063,45 @@ def is_deriv_k(fa, fd, DE):
             return (ans, result, const)
 
 
+def base_Q(G):
```

Comment:
Can you just use the "parallel" constructor in the polys to do this work for you?